steampipecloud/table_steampipecloud_audit_log.go

```diff
@@ -2,10 +2,7 @@ package steampipecloud
 
 import (
 	"context"
-	"net/http"
-	"time"
 
-	"github.com/sethvargo/go-retry"
 	openapi "github.com/turbot/steampipe-cloud-sdk-go"
 
 	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
```

Comment:
For this table, are we able to list audit logs for all user and organization workspaces, which would then allow us to remove the required key qual (similar to the workspace and workspace_connection tables)? Also, are we able to add more optional list key quals here, namely `identity_handle` and `identity_type`?

Reply:
@cbruno10 we do not have an API that will list all audit logs(user + org) like workspace or connection. For audit log we have separate API's for user and org. Also, `identity_handle` must be passed to those API. `identity_type` could not be an optional qual as API's does not support it.